reference/machine-migration.html.markerb

```diff
@@ -0,0 +1,83 @@
+---
+title: Machine migration
+layout: docs
+nav: firecracker
+---
+
+Fly.io will migrate your Machines from problematic hosts to healthy hosts as required. Some of the reasons for migrating Machines include host overcrowding, deprecation, or scheduled maintenance. The migration process helps your applications run efficiently on the Fly.io platform. You don't need to take any action to migrate your Machines to another host and, if your app has multiple Machines, migration shouldn't result in downtime for your app.
+
+## Machine migration process
+
+The Machine migration process consists of several steps:
+
+1. Acquire an internal lock to coordinate multiple Machine migrations in the Fly.io platform globally.
+1. Stop a Machine.
+1. Fork the Machine's volume to the destination host (if the Machine has an attached volume).
+1. Start a new Machine on the destination host.
+1. Release the internal lock.
+
+The following sections describe the process in more detail.
+
+### 1. Acquire an internal lock
+
+The internal lock for migration is per network ID. All the apps in your organization are on a [private network](/docs/networking/private-networking/) by default. There's no way around it, a Machine will be stopped prior to migration. When we lock by network ID, we ensure that we don't migrate other Machines from the same app or, in fact, any other apps in your network, at the same time. This can prevent clustering or quorum issues for databases and avoid downtime for apps with multiple Machines.
+
+### 2. Stop a Machine
+
+We send the Machine the signal configured as its `kill_signal` in `fly.toml` or Machine config and then shut down. This should result in a graceful shutdown of a running app Machine.
+
+
+### 3. Fork the Machine's volume
+
+Things to cover
+
+- Volume IDs will be different
+
+### 4. Start a new machine
+
+Things to cover
+
+- Machine IDs will be same
+
+### 5. Release the internal lock
+
+## Check whether your machine was migrated
+
+The `flyctl machine status` command shows migration events in the Event Logs. For example, the following Machine was migrated, as indicated the `import=true` info in the `created` event log:
+
+```cmd
+ go run . m status -a epic-stack-691c
```

Comment:
We need to use `flyctl` here instead of `go run`.

Reply:
yep ty! I made a few more edits to the output